README.md

```diff
@@ -79,6 +79,7 @@ zoomSrc | String | | URL for the larger zoom image. Falls back to original image
 zoomScale | Number | 1 | Multiplied against the natural width and height of the zoomed image. This will generally be a decimal (example, 0.9 for 90%).
 zoomPreload | Boolean | false | If set to true, preloads the zoom image instead of waiting for mouseenter and (unless on a touch device) persists the image on mouseleave.
 alt | String | | Alternative text for the original image.
+title | String | | The imagem title.
```

Comment:
This might be a little confusing since it doesn't follow the same pattern as the other prop descriptions. Maybe try something like "Title attribute for the original image." instead?

Reply:
Thank you so much for this lib and your attention! I fixed the title description as you suggested in dd5bbcb.
And about the ```imgAttributes``` feature, i'd be glad to do it if you let met :smile:

Reply:
Merged! And I'd love the help, I'll try to take a look later this week and write up some details either in issues or discussions.